src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/CodeReaderWriter/ProjectWorkspace.cs

```diff
@@ -0,0 +1,37 @@
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.MSBuild;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.MSIdentity.CodeReaderWriter
+{
+    public class CodeAnaylsisHelper
```

Comment:
>CodeAnaylsisHelper [](http://example.com/codeflow?start=17&length=18)

file name is ProjectWorkspace - shoudl we rename it?

Reply:
fixed.